modules/kn-service-apply.adoc

```diff
@@ -0,0 +1,36 @@
+// Module included in the following assemblies:
+//
+// * serverless/knative_serving/serverless-applications.adoc
+// * serverless/cli_reference/kn-services-ref.adoc
+
+[id="kn-service-apply_{context}"]
+= Applying service declarations
+
+You can declaratively configure a Knative service by using the `kn service apply` command. If the service does not exist it will be created, otherwise the existing service will be updated with the options that have been changed.
+
+The `kn service apply` command is especially useful when used in an automated way, such as in a shell script or a continuous integration pipeline, because in these types of applications users typically want to fully specify the state of the service in a single command to declare the target state.
+
+When using `kn service apply` you must provide the full configuration for the Knative service. This is different to the `kn service update` command, which only requires you to specify in the command the options that you want to update.
+
+.Example commands
+
+* Create a service:
++
+[source,terminal]
+----
+$ kn service apply <service_name> -i <image>
```

Comment:
With the version from Serverless 1.14, I get

```
kn service apply hello -i gcr.io/knative-samples/helloworld-go
Error: unknown shorthand flag: 'i' in -i for 'kn service apply'
Run 'kn service apply --help' for usage
```

`--image` works

Reply:
Thanks, I'm trying to keep flags consistent in the docs and generally we use the short version where possible but it's good to know it's not an option here. Will update.